shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagerSyncService.java

```diff
@@ -0,0 +1,23 @@
+package com.redhat.service.smartevents.shard.operator.v2;
+
+import io.quarkus.scheduler.Scheduled;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class ManagerSyncService {
```

Comment:
@vaibhavjainwiz We need a unit test for this please.

Reply:
done